### PR TITLE
update for certbot instructions

### PR DESCRIPTION
**Pull request by SwartzCr**

### CODE REVIEW

**Comment by SwartzCr** [comment]
@pde or @hlieberman plz review

File: _scripts/instruction-widget/install.js [hunk 1]
```diff
@@ -84,16 +85,24 @@ module.exports = function(context) {
...
     if (context.distro == "debian" && context.version == 8) {
       context.backports_flag = "-t jessie-backports";
     }
 
     // Set package based on webserver.
-    // TODO: I don't think these packages are correct
     if (context.webserver == "apache") {
       context.package = "python-letsencrypt-apache";
     }
+    if (context.id == "debiantesting") {
```
Comment (bmw) [inline _scripts/instruction-widget/install.js]: Is this supposed to be OS?

File: _scripts/instruction-widget/install.js [hunk 2]
```diff
@@ -84,16 +85,24 @@ module.exports = function(context) {
...
+      if (context.webserver == "apache") {
+        context.package = "python-certbot-apache";
+      }
+      context.package = "python-certbot"
```
Comment (bmw) [inline _scripts/instruction-widget/install.js]: This will override setting package to `python-certbot-apache` in the body of the `if` statement above.

**Comment by bmw** [comment]
Also, `certbot` isn't in Debian testing yet according to [this](https://packages.debian.org/search?keywords=certbot&searchon=names&suite=testing&section=all). Other than that and my comments above, this LGTM.

File: _scripts/instruction-widget/install.js [hunk 3]
```diff
@@ -97,11 +97,12 @@ module.exports = function(context) {
     if (context.webserver == "apache") {
       context.package = "python-letsencrypt-apache";
     }
-    if (context.id == "debiantesting") {
+    if (context.os == "debianunstable") {
+      context.base_command = "certbot";
+      context.package = "python-certbot"
```
Comment (bmw) [inline _scripts/instruction-widget/install.js]: nit: Based on [these descriptions](https://packages.debian.org/search?keywords=certbot&searchon=names&suite=unstable&section=all) I think we should tell people to install `certbot` (plus it's a shorter package name).

**Comment by bmw** [comment]
LGTM. Will merge when Travisi s green.

**Pull request merged**